content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/clientlibs/site/js/carousel.js

```diff
@@ -59,6 +59,19 @@
                 value = parseFloat(value);
                 return !isNaN(value) ? value : null;
             }
+        },
+        /**
+         * Determines whether automatic pause on hovering the carousel is disabled
+         *
+         * @memberof Carousel
+         * @type {Boolean}
+         * @default false
+         */
+        "autopauseDisabled": {
+            "default": false,
+            "transform": function(value) {
+                return !(value === null || typeof value === "undefined");
```

Comment:
What if `value` is `false`?

Reply:
@vladbailescu - this is a boolean attribute [0]. The presence of the `data-autopause-disabled` attribute indicates `true`, the attribute missing indicates `false`.

[0] https://www.w3.org/TR/html4/intro/sgmltut.html#h-3.3.4.2